carma_wm/src/WMListenerWorker.h

```diff
@@ -115,6 +126,7 @@ class WMListenerWorker
   std::function<void()> route_callback_;
   void newRegemUpdateHelper(lanelet::Lanelet parent_llt, lanelet::RegulatoryElement* regem) const;
   double config_speed_limit_;
+  std::string participant_;
```

Comment:
I don't think you need to actually store the participant_ in the WMListenerWorker just forward it to the world model object.